src/libNode/FinalBlockProcessing.cpp

```diff
@@ -351,7 +351,8 @@ bool Node::CheckMicroBlockRootHash(const TxBlock& finalBlock,
 }
 
 #ifndef IS_LOOKUP_NODE
-void Node::BroadcastTransactionsToLookup()
+void Node::BroadcastTransactionsToLookup(
```

Comment:
This doesn't seem to be called by the DS node, only by the shard nodes after final block is received